### DISCUSSION TITLE
stacked polar chart for windrose

### DISCUSSION THREAD

**Discussion by lionpeloux** [apexcharts/apexcharts.js#3588] [Q&A]
Apexcharts is great ... but ...

I've read the docs thoroughly and could not find a way to make a "stacked polar chart" : 
* is this possible ?
* Are there any plans to add this behaviour to the existing polar chart ?

I think a classic polar chart should behave more or less as a bar (eventually stacked).
[a javascript example here](https://jscharting.com/examples/chart-types/radar-polar/wind-rose/)

<img width="678" alt="winrosejs" src="https://user-images.githubusercontent.com/951626/210956343-7a5cd76e-5fd4-42ca-88c6-096bda48f0b4.png">

**Comment by anastasiyaArk** [comment]
It has been a while since you've asked this question. Have you ever found it out? I have a vue3 app and consider using Apexcharts

**Reply by lionpeloux** [reply]
Sorry but, I did not go further on this topic. If you find any option you are welcome to share it ;).
